lsp/basedpyright.lua

```diff
@@ -42,10 +42,12 @@ return {
   },
   on_attach = function(client, bufnr)
     vim.api.nvim_buf_create_user_command(bufnr, 'LspPyrightOrganizeImports', function()
-      client:exec_cmd({
+      local params = {
         command = 'basedpyright.organizeimports',
         arguments = { vim.uri_from_bufnr(bufnr) },
-      })
+      }
```

Comment:
not sure why you changed this (always need to mention that in the pr), but if your intention is to pass bufnr, that can be done as the 2nd arg to exec_cmd

    client:exec_cmd({ ... }, { bufnr = bufnr })

Reply:
@justinmk It's explained in the PR description and commit msg: `client:exec_cmd()` was raising an error about that command not being supported by the server and aborting, so I got the old version from [here](https://github.com/neovim/nvim-lspconfig/blob/f47cd681d7cb6048876a2e908b6d8ba1e530d152/lua/lspconfig/configs/pyright.lua#L24) but I assumed you needed to pass `bufnr` instead of `0`; it works with `0` too, but it doesn't work if you change it back to `client:exec_cmd()`.

Reply:
> Did you pass bufnr in the 2nd dict arg as I suggested?

```lua
      client:exec_cmd({
        command = 'pyright.organizeimports',
        arguments = { vim.uri_from_bufnr(bufnr) },
      }, {bufnr = bufnr})
```

This raises the same error.

Reply:
? `client:exec_cmd` performs `"workspace/executeCommand"`. The only difference is that  `exec_cmd` currently is [always asynchronous](https://github.com/neovim/nvim-lspconfig/pull/3876) (but so is `client.request`).

So, there is some other problem, and the change you've made here only accidentally "fixes" a symptom. Need to find out what the actual problem is.

Reply:
Let's wait a bit and see if others have comments.